src/tests/Microsoft.PowerFx.Connectors.Tests.Shared/Helpers/LoggingTestServer.cs

```diff
@@ -160,8 +160,15 @@ protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage
                         _log.AppendLine($" [content-header] {h.Key}: {string.Join(", ", h.Value)}");
                     }
                 }
+                
+#if NET7_0_OR_GREATER
+                var content = await httpContent.ReadAsStringAsync(cancellationToken);
+#else
+
+                // We cannot pass the cancellation token in .Net 4.6.2
+                var content = await httpContent.ReadAsStringAsync();
```

Comment:
>ReadAsStringAsync [](http://example.com/codeflow?start=48&length=17)

one trick we can use here is "shims". 
Ie, for !7.0, we define an extension method that accepts the cancellation token. 

This lets us centralize the issues and reduce number of #ifs.

Reply:
I'll do that